### PR TITLE
New issue 1271499 remove about me

### PR DESCRIPTION
**Pull request by emullaney**
For https://bugzilla.mozilla.org/show_bug.cgi?id=1271499
* Remove About Me (bio) from Profile display/edit pages
* Remove Website URL from Profile display/edit pages
* Adjust tests for these 2 fields being removed from pages

### CODE REVIEW

File: kuma/users/jinja2/users/user_detail.html [hunk 1]
```diff
@@ -106,11 +102,6 @@ <h1 class="user-title">
           <img src="{{ gravatar_url(detail_user.email, size=200) }}" alt="{{ detail_user.username }}" width="200" height="200" class="user-photo avatar" />
         </figure>
 
-        <ul class="user-links">
```
Comment (jwhitlock) [inline kuma/users/jinja2/users/user_detail.html]: This ``<ul>`` should be retained for the rest of the links

File: kuma/users/forms.py [hunk 2]
```diff
@@ -122,7 +113,7 @@ class UserEditForm(forms.ModelForm):
     class Meta:
         model = User
         fields = ('fullname', 'title', 'organization', 'location',
-                  'locale', 'timezone', 'bio', 'irc_nickname', 'interests',
+                  'locale', 'timezone', 'irc_nickname', 'interests',
                   'website_url', 'twitter_url', 'github_url',
```
Comment (jwhitlock) [inline kuma/users/forms.py]: ``'website_url'`` should be dropped as well.

**Comment by jwhitlock** [comment]
Practical tests work:
* The "About Me" section and the website URL are gone from the profile (with display issue due to dropped `<ul>` tag)
* They are also gone when editing my profile
* The profile can be saved successfully without them

There's a few issues noted above, plus some more items to remove:
* [ ] [kuma/users/admin.py](https://github.com/mozilla/kuma/blob/master/kuma/users/admin.py#L28), ``UserAdmin`` - remove ``bio`` and ``website`` from ``list_display``, ``bio`` and ``homepage`` from the ``search_fields``, and drop the ``website()`` method.
* [ ] [kuma/users/tests/test_views.py](https://github.com/mozilla/kuma/blob/master/kuma/users/tests/test_views.py#L125-L137), ``UserViewsTest._get_current_form_field_values``, drop ``bio``
* [ ] [kuma/static/styles/components/users/user-head.styl](https://github.com/mozilla/kuma/blob/master/kuma/static/styles/components/users/user-head.styl#L54-L56), drop the ``.user-bio`` rule
* [ ] [kuma/users/tests/test_forms.py](https://github.com/mozilla/kuma/blob/master/kuma/users/tests/test_forms.py#L65-L80), ``TestUserEditForm.test_https_user_urls`` - drop ``website`` from test set.

File: kuma/users/jinja2/users/user_detail.html (hunk 1 above)
Comment (emullaney) [inline kuma/users/jinja2/users/user_detail.html]: Ah, I knew it looked odd but didn't catch this. Thanks.

**Comment by emullaney** [comment]
Thanks very much for the thorough feedback. This should be all set and ready for re-review.

[x] kuma/users/admin.py, UserAdmin - remove bio and website from list_display, bio and homepage  from the search_fields, and drop the website() method.
[x] kuma/users/tests/test_views.py, UserViewsTest._get_current_form_field_values, drop bio
[x] kuma/static/styles/components/users/user-head.styl, drop the .user-bio rule
[x] kuma/users/tests/test_forms.py, TestUserEditForm.test_https_user_urls - drop website from test set.

**Comment by jwhitlock** [comment]
Looks good. Good-bye bio and website!

**Pull request merged**